app/assets/javascripts/qz/ticket-popup.js

```diff
@@ -0,0 +1,85 @@
+var privateKey = "-----BEGIN RSA PRIVATE KEY-----\n" +
```

Comment:
Yeah I'm totally breaking their "security model". But I have good reasons to do so.
First the "attack surface" is narrow. Someone willing to print a ticket would launch the qz client (the one I compiled for OFN) and, at his first print, would accept the certificate (cert.pem) as trusted. The bad scenario is that a bad guy would trick this person to navigate to a page where he would have used the (in fact public) private key to well, print things via qz.
It's worth noting that this scenario is probably not the only reason for the existence of the cryptosystem, as it's also what they use for their business model: companies would pay for a key pair generated for the cert inside the release of the client that they distribute. They still don't absolutely enforce it as they [allow to bundle you own certificate](https://qz.io/wiki/compiling#ant-command-line).
And having a private key that needs to be both kept private and distributed to anyone willing to deploy OFN is something that seems to me rather hard or at least pretty impratical.
So I believe it's not worth it.